Dockerfile

```diff
@@ -77,17 +77,20 @@ RUN cd ./deploy/tools/envs-validator && yarn build
 # ******* STAGE 3: Run ********
 # *****************************
 # Production image, copy all the files and run next
-FROM node:18-alpine AS runner
+FROM node:18-alpine3.18 AS runner
 RUN apk add --no-cache --upgrade bash curl jq unzip
 
 ### APP
 WORKDIR /app
 
+ENV NODE_ENV production
 # Uncomment the following line in case you want to disable telemetry during runtime.
-# ENV NEXT_TELEMETRY_DISABLED 1
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN deluser --remove-home node && delgroup node
 
-RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
+RUN addgroup nonroot --gid 1100 && \
+  adduser nonroot --ingroup nonroot --uid 1100 --home /home/nonroot --disabled-password
```

Comment:
Can you explain please why this approach is better than the current one which was taken from Next.js docs?

Do we really need to delete the `node` user?